gtest-parallel

```diff
@@ -168,11 +183,9 @@ class FilterFormat:
     stdout_lock.release()
 
   def end(self):
-    if self.failures:
-      self.out.permanent_line("FAILED TESTS (%d/%d):"
-                              % (len(self.failures), self.total_tests))
-      for (binary, test) in self.failures:
-        self.out.permanent_line(" " + binary + ": " + test)
+    self.print_tests("FAILED TESTS", self.failures)
+    interruptions = set(self.started).difference(set(self.finished))
+    self.print_tests("INTERRUPTED TESTS", interruptions)
```

Comment:
1. Don't we already print failed tests?
2. When you print lists of tests, would it make sense to sort them first?